payments-core-testing/build.gradle

```diff
@@ -12,6 +12,11 @@ dependencies {
     implementation testLibs.mockito.kotlin
     implementation testLibs.turbine
     implementation testLibs.leakCanaryInstrumentation
+    implementation project(':paymentsheet')
```

Comment:
I wonder if we should create a new module for paymentsheet testing. Adding everything to payments core spreads things out. 

I think the answer is we want to, but I think it's reasonable to throw things in here for now, and tease them out later.

Reply:
https://github.com/stripe/stripe-android/pull/10318